mlflow/shap.py

```diff
@@ -433,7 +434,7 @@ def save_explainer(
     explainer_data_subpath = "explainer.shap"
     explainer_output_path = os.path.join(path, explainer_data_subpath)
     with open(explainer_output_path, "wb") as explainer_output_file_handle:
-        explainer.save(explainer_output_file_handle)
+        explainer.save(explainer_output_file_handle, model_saver=model_saver)
```

Comment:
When will the new SHAP Serialization API be relased? Next release?

Reply:
It was released yesterday https://github.com/slundberg/shap/releases/tag/v0.39.0

Reply:
This change to the API has back-compat for the API but not for the actual artifacts generated by saving.